Tests/test_file_tiff.py

```diff
@@ -450,6 +450,18 @@ def test_save_tiff_with_jpegtables(self):
         # Should not raise UnicodeDecodeError or anything else
         im.save(outfile)
 
+    def test_open_tiff_uint16_multiband(self):
+        """Test opening multiband TIFFs and reading all channels."""
+        base_value = 4660
+        for i in range(2, 6):
+            infile = "Tests/images/uint16_{}_{}.tif".format(i, base_value)
+            im = Image.open(infile)
+            im.load()
+            for j in range(i):
+                current_value = base_value + j
+                im.seek(j)
+                self.assertEqual(im.getpixel((0, 0)), current_value)
```

Comment:
Yeah I figured, but I thought it was a start. Is there a way to signify greyscale bands in `TiffImagePlugin.py:OPEN_INFO` or is this going to require some more extensive changes?